docs/about_Puppet.Dsc.md

```diff
@@ -0,0 +1,70 @@
+This means you are not responsible for shipping any external code to your managed nodes - as with other Puppet modules, everything you need is "in the box," either in the generated module or the [Puppet pwshlib module][pwshlib] on which it depends.
+
+This also means that you don't have to manage any additional versions or worry about cross-version contamination: Puppet is only and always going to use the vendored resources, not whatever else may or may not be on the target node!
+
+## Updating the Puppet module
+
+This module takes care of updating the Puppet module with information from the PowerShell module:
+
+- It writes a README for the Puppet module which stipulates information, dependencies, and points both to this PowerShell module and the one it has vendored.
+- It updates the Puppet module fixtures, ensuring that if all you have is the PDK and this PowerShell module, you can set up and functionally test the Puppetized module.
+- It updates the Puppet module's metadata, pinning the version of the Puppet module to match the PowerShell module and inserting some useful metadata for troubleshooting.
+  It also sets the issues and project URLs *not* to Puppet-owned links, but to the upstream PowerShell module's settings.
+
+## Converting the DSC resources to Puppet types
+
+The heavy lifting for this module happens in the `ConvertTo-PuppetResourceApi` function, which introspects on DSC resources, parses them, and creates a Puppetized representation of those resources.
+
+The architecture of this process is described more in depth in [about_Puppetization][about_Puppetization].
+
+## Writing the types and providers
+
+The module uses a few small functions which include here-strings to to write the appropriate files.
+`Get-TypeContent` takes the information parsed in the `ConvertTo-PuppetResourceApi` function and uses it to write the Puppet Resource API type file, which includes all of the appropriate metadata, parameters, and documentation.
+The `Get-ProviderContent` function fills out a very small provider file; this is because all of the actual provider functionality lives in the [`dsc_base_provider`](base_provider) in the [**puppetlabs/pwshlib**](pwshlib) module.
+The provider written in the Puppetized module is merely an inheritor of that base provider with the appropriate name.
+
+## Writing the reference documentation
+
+Finally, the module uses the PDK to fill out all of the appropriate reference documentation based on the type files from the prior step.
+This ensures that the reference docs show up on the forge and in a single markdown document that you can review if you download the Puppetized module.
+
+## Finally
+
+All of those things done, the Puppetizer has vendored and converted a PowerShell module with DSC Resources into a Puppet module, including all of the appropriate documents and metadata for troubleshooting and use.
```

Comment:
`With all of these actions completed`.

Also you use the word 'Puppetizer' here, but nowhere else in the document. Is this the name of the builder?

Reply:
Ach, this is a good point; I need to clarify and settle on the terminology for this overall, because the _module_ is `puppet.dsc` and I think the appropriate (non-canonical) verb is `puppetize` but maybe another word works better here?